examples/templates/aws-devcontainer/main.tf

```diff
@@ -144,24 +216,57 @@ locals {
     echo "Docker is already installed."
   fi
 
+  # Set up Docker credentials
+  mkdir -p "/home/${local.linux_user}/.docker"
+  if [ -n "${local.docker_config_json_base64}" ]; then
+     # Write the Docker config JSON to disk if it is provided.
+     printf "%s" "${local.docker_config_json_base64}" | base64 -d | tee "/home/${local.linux_user}/.docker/config.json"
+  else
+    # Assume that we're going to use the instance IAM role to pull from the cache repo if we need to.
+    # Set up the ecr credential helper.
+    apt-get update -y && apt-get install -y amazon-ecr-credential-helper
```

Comment:
Does it work without configuring a repo? I mean is it available in default Ubuntu repos?

Reply:
yep